src/lib/conf/conftypes.h

```diff
@@ -110,6 +110,16 @@ typedef struct struct_magic_decl_t {
  * fetch this option should produce a warning.
  **/
 #define CVFLAG_OBSOLETE  (1u<<0)
+/**
+ * Flag to indicate that an option is undumpable. An undumpable option is
+ * never saved to disk, and is prefixed with __.
```

Comment:
Why does this comment say __, but the next comment not say ___?

Reply:
I've tried to clarify this in cde662e2590058, and mark both prefixes as historical.